DXMainClient/DXGUI/Generic/TopBar.cs

```diff
@@ -224,10 +224,17 @@ public override void Initialize()
             connectionManager.WelcomeMessageReceived += ConnectionManager_WelcomeMessageReceived;
             connectionManager.AttemptedServerChanged += ConnectionManager_AttemptedServerChanged;
             connectionManager.ConnectAttemptFailed += ConnectionManager_ConnectAttemptFailed;
+            connectionManager.ErrorDroneBL += ConnectionManager_ErrorDroneBL;
 
             privateMessageHandler.UnreadMessageCountUpdated += PrivateMessageHandler_UnreadMessageCountUpdated;
         }
 
+        private void ConnectionManager_ErrorDroneBL(object sender, string message)
+        {
+            XNAMessageBox messageBox = new XNAMessageBox(WindowManager, "Error connecting", message, XNAMessageBoxButtons.OK);
+            messageBox.Show();
```

Comment:
You should be able to just do:
`XNAMessageBox.Show(WindowManager, "Error connecting", message);`

Reply:
Agreed, should do this instead.